### PR TITLE
Ftr/3.0msgpack and triple codec extension samples

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**

### CODE REVIEW

File: general/dubbo3/codec-extension/go-server/codec/codec.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package codec
+
+import (
+	"encoding/json"
+	triCommon "github.com/dubbogo/triple/pkg/common"
```
Comment (zouyx) [inline general/dubbo3/codec-extension/go-server/codec/codec.go:5]: split？

**Review by zouyx — APPROVED** [review]

**Comment by AlexStocks** [comment]
Pls change Id to ID.

File: general/dubbo3/codec-extension/go-server/codec/codec.go (hunk 1 above)
Comment (LaurenceLiZhixin) [reply general/dubbo3/codec-extension/go-server/codec/codec.go:5]: fixed

**Comment by LaurenceLiZhixin** [comment]
> 
> 
> Pls change Id to ID.

fixed

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**